fedoracommunity/search/index.py

```diff
@@ -111,51 +117,54 @@ def create_index(self):
     @property
     def latest_release(self):
         if not self._latest_release:
-            response = local.http.get(self.pkgdb_url + "/api/collections")
-            if not bool(response):
-                raise IOError("Unable to find latest release %r" % response)
-            data = response.json()
-
-            # Strip off rawhide
-            data['collections'] = [
-                c for c in data['collections'] if c['version'] != 'devel'
-            ]
+            releases_all = self.get_all_releases_from_bodhi()
```

Comment:
Yeah, let's just avoid this for now.  The right thing to do will be to query PDC for the releases... but I'd rather just punt for the time being (fedora-packages is non-critical).